Backend/HTML/verified.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<head>
+
+  <meta charset="utf-8">
+
+  <title>HEARYOU Email verification</title>
+
+</head>
+
+  <body style="margin: 0em">
+
+  <header style="color: #fff; text-align: center; background-color: #18BC9C">
```

Comment:
Please be consistent with color hex code length - if you want to use 6 digits, then update #fff to #ffffff